config.py

```diff
@@ -1201,6 +1201,26 @@
 confidence interval of the mean decoding score.
 """
 
+csp_reg: float = 0.1
+"""
+Regularization used in the covariance estimator when calculating CSPs.
+"""
+
+cluster_stats_alpha: float = 0.05
+"""
+Statistic level used in the time frequency script.
+"""
+
+cluster_stats_alpha_t_test: float = 0.1
+"""
+Threshold level used to determine clusters.
+"""
```

Comment:
I would name this param differently, e.g.,

```
cluster_t_dist_alpha_thres: float = 0.1
"""
The percentile of the t-distribution to use as the threshold when creating the
clusters. The t-distribution is created with `number of subjects - 1` degrees
of freedom. For example, if you set this value to `0.05` and your analysis
includes 10 subjects, the value of the 95th percentile of the t-distribution
`t(10)` will be used for thresholding the clusters.
"""
```
@agramfort WDYT?

Reply:
in all cases, your description is way better than mine